### PR TITLE
Increment docs

### PR DESCRIPTION
**Pull request by ryan-shea**
#### Problem

Instead of using dev time to change text in documentation files, let's use the power of the computer to do it for us.

#### Summary of Changes

Added a second movement to `increment-cargo-version.sh`

### CODE REVIEW

File: scripts/increment-cargo-version.sh [hunk 1]
```diff
@@ -108,6 +109,15 @@ for Cargo_toml in "${Cargo_tomls[@]}"; do
   done
 done
 
+# Update all the documentation references
+for Doc_update in "${Doc_updates[@]}"; do
+  # Set new crate version
+  (
+    set -x
+    sed -i "$Doc_update" -e "s/v[[:digit:]]\+\.[[:digit:]]\+\.[[:digit:]]/$newVersion/g"
```
Comment (mvines) [inline scripts/increment-cargo-version.sh]: We only want to replace `currentVersion` with `newVersion`, not all occurrences of x.y.z

**Comment by garious** [comment]
@ryan-shea, can you wrap this one up?

**Comment by ryan-shea** [comment]
@garious @mvines thanks for the update — will get to this this weekend.

I believe the only instances of x.y.z. are with `currentVersion`. 

If I want to find and replace `currentversion`, is it just a matter of changing:

`sed -i "$Doc_update" -e "s/$currentVersion/$newVersion/g"`

**Comment by mvines** [comment]
Yes I think that’ll do it

**Comment by ryan-shea** [comment]
@mvines is this ready to squash? I think it should do the job

File: scripts/increment-cargo-version.sh [hunk 2]
```diff
@@ -108,6 +110,15 @@ for Cargo_toml in "${Cargo_tomls[@]}"; do
   done
 done
 
+# Update all the documentation references
+for Doc_update in "${Doc_updates[@]}"; do
```
Comment (mvines) [inline scripts/increment-cargo-version.sh]: nit: let's rename these variables to something a little more clear:
I suggest `Doc_updates` -> `markdownFiles`, and `Doc_update` -> `file`

**Comment by ryan-shea** [comment]
@mvines think this should be good to go

**Comment by mvines** [comment]
lgtm, CI is backlogged right now but once it's happy then lets merge

**Comment by solana-grimes** [comment]
:broken_heart: Unable to automerge due to CI failure

**Comment by mvines** [comment]
@ryan-shea - CI failed because you PR is based on a pretty old commit off master.  If you rebase to the latest on master we should be gtg

**Comment by ryan-shea** [comment]
uhhh... don't know what I did

https://github.com/solana-labs/solana/pull/6456 here's a fresh one

**Pull request closed without merging**